web/src/read_receipts.js

```diff
@@ -79,9 +78,5 @@ export function show_user_list(message_id) {
                 });
             }
         },
-        on_hide() {
-            // Ensure any user info popovers are closed
-            popovers.hide_all();
-        },
```

Comment:
I'm not sure this one is safe to remove -- is it possible for the read receipts modal to close in the background, without a click that would hide these popovers? Though I guess Tippy is smart enough to hide popovers when their reference is removed from the DOM?